content/docs/command-reference/get.md

```diff
@@ -36,16 +36,16 @@ the data source. Both HTTP and SSH protocols are supported for online repos
 to an "offline" repo (if it's a DVC repo without a default remote, instead of
 downloading, DVC will try to copy the target data from its <abbr>cache</abbr>).
 
+⚠️ Online repos should have a default
```

Comment:
it's not only about online repos though

Reply:
Here's the whole idea (text fragments from the paragraph above):

> Both HTTP and SSH protocols are supported for online repos
> `url` can also be a local file system path
> Online repos should have a default DVC remote

I'm not sure what is missing. Or are you agains the term "online repo"? We could come up with another way of saying it.

Reply:
Yeah I guess the whole paragraph is too complicated, let me see how to re-write it and avoid terms online/offline.

Reply:
OK done in e6359056. PTAL

Reply:
yep, LGTM!